src/pages/api/sepana/deleteAll.page.ts

```diff
@@ -0,0 +1,29 @@
+import { NextApiHandler } from 'next'
+import { deleteAllSepanaDocs, sepanaAlert } from './utils'
+
+const handler: NextApiHandler = async (_, res) => {
+  try {
+    await deleteAllSepanaDocs()
+  } catch (error) {
+    await sepanaAlert({
+      type: 'alert',
+      alert: 'DB_UPDATE_ERROR',
+      body: JSON.stringify(error),
+    })
+
+    res.status(500).json({
```

Comment:
**question** does this need a `return` statement?

Reply:
handlers should be void afaik, important thing is ending the request